src/database/db_manager.py

```diff
@@ -18,41 +29,74 @@ def __init__(self, idx=None, message=None):
         super().__init__(self.message)
 
 
-class DummyDBManager:
+class DatabaseManager:
     def __init__(self, api_manager):
-        self.users = []
+        self.logger = logging.getLogger(__name__)
+
+        self.DB_FOLDER = getenv("DB_FOLDER")
+        if self.DB_FOLDER is None or not path.isdir(self.DB_FOLDER):
+            self.logger.critical("DB_FOLDER: '%s', is not a directory", self.DB_FOLDER)
+            raise Exception(f"DB_FOLDER: '{self.DB_FOLDER}', is not a directory")
+
+        # Init Connection object allowing interaction with the database
+        db_file_path = path.join(self.DB_FOLDER, DB_FILE_NAME)
+        self.db = sqlite3.connect(db_file_path)
```

Comment:
Don't we need a `try except` here to detect the failed connection and write it in the logs ?

Reply:
Well in fact, this is similar to a file opening so it should not fail. Moreover, if it does, it should rather crash to easily know something went wrong.